QuasarHelios2015/src/org/team1540/quasarhelios/HeadingSensor.java

```diff
@@ -32,6 +39,26 @@ public static void setup() {
 
         zeroGyro = sensor.zeroGyro;
 
+        absoluteYaw = FloatMixing.addition.of(FloatMixing.multiplication.of((FloatInput) accumulator, 360), yaw);
+
+        Igneous.globalPeriodic.send(new EventOutput() {
+            public void event() {
+                float currentyaw = yaw.get();
+                float oldyaw = oldYaw.get();
+                if (Math.abs(currentyaw - oldyaw) > 180) {
+                    if (oldyaw > 180) {
+                        accumulator.set(accumulator.get() + 1);
+                    } else if (oldyaw < -180) {
+                        accumulator.set(accumulator.get() - 1);
+                    }
+                }
+                oldYaw.set(yaw.get());
+            }
+        });
+
+        Cluck.publish("Accumulator", accumulator);
+        Cluck.publish("Absolute Yaw", absoluteYaw);
```

Comment:
Maybe these names could be qualified better to make it clear what they're talking about.